src/components/DataAccessRequest.js

```diff
@@ -0,0 +1,57 @@
+import _ from 'lodash';
+import { Component } from 'react';
+import { div, hh, span } from 'react-hyperscript-helpers';
+
+
+const parseAlias = (alias) => {
```

Comment:
alternative function body: 

```
const prefix = 'DUOS-'; 
const pad = '000000'; 
return  prefix + (pad+n).slice(-pad.length);
```

Reply:
Interesting approach - will investigate. I originally stole this from https://github.com/DataBiosphere/consent/blob/3d8cd299580ce729d1cd57652843ef9b8448047d/src/main/java/org/broadinstitute/consent/http/util/DatasetUtil.java#L7

Reply:
Done!